packages/neptune-extension/config/webpack.common.js

```diff
@@ -73,6 +73,8 @@ module.exports = function ({ mode, platform, project }) {
        */
       new webpack.DefinePlugin({
         'NEPTUNE_BUILD_DATE': JSON.stringify(new Date()),
+        'PLATFORM': JSON.stringify(platform),
```

Comment:
Correct me if I am wrong but I see no usage of this variable. Is there a reason to introduce this variable ?

Reply:
Now I see, this variable can be used to check for upgrade in nbextension only.

Reply:
Yep. That was the purpose.
I did it in this PR because it was just a few seconds of overhead to this PR.